src/commcare_cloud/fab/fabfile.py

```diff
@@ -221,7 +221,7 @@ def env_common():
     all = servers['all']
     proxy = servers['proxy']
     webworkers = servers['webworkers']
-    manage = webworkers[0]
+    manage = servers.get('manage', webworkers[0])
```

Comment:
Is the idea here that the inventory file can specify a preferred "manage" machine?

Reply:
Yeah, emord's [suggestion](https://github.com/dimagi/commcare-cloud/pull/1651/files/5d8d4bb11edfbacb0f70e5647ab3556860dbc045#diff-4b38b47cd9db17e5e8e82f5424318f83] since on icds apparently webworkers:0 is already a bit over taxed

Reply:
I like the pattern. For it to play nice with our other tools, this should be the default for the `DjangoManage` and `Tmux` commands as well